src/app/components/dashboard/dashboard.component.html

```diff
@@ -3,7 +3,10 @@
 <!-- main -->
 <div class="container">
   <div class="alert alert-warning alert-dismissible fade show" role="alert">
-    <h3 style="color: black;"><strong>Welcome {{Username}}!👋</strong></h3>
+    <h3 style="color: black;
```

Comment:
please define all the css in a class and just use that class here instead of inline styling.